Introduction/ChoosingODEAlgorithm.ipynb

```diff
@@ -1306,7 +1306,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's zoom in on the y-axis to see what's going on:"
+    "Let's zoom in on the horizontal axis to see what's going on:"
```

Comment:
it is the y-axis because it's just done by limiting `y`.

Reply:
fixed it!